__ratingsTests__/ReviewList.test.js

```diff
@@ -0,0 +1,226 @@
+import React from 'react';
+import { cleanup, render, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { afterEach, describe, it, expect } from '@jest/globals';
+import jest from 'jest';
+import ReviewList from '../client/src/components/Sub_RatingsReviews/ReviewList';
+
+afterEach(cleanup);
+
+describe('More Reviews', () => {
```

Comment:
- I can't provide much useful commentary since I am still needing to catch up on my test knowledge, but that considered, what I see looks sensible